WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -216,6 +218,15 @@ protected SendControlViewModel(Wallet wallet, string title)
 				{
 					AmountText = url.Amount.ToString(false, true);
 				}
+
+				if (url.UnknowParameters.TryGetValue("bpu", out var endPoint))
```

Comment:
This may change from `bpu` to `pj`

Reply:
In general I think we should stick with `payjoin` over `bustapay`.

Reply:
Yeah, considering that we've deviated quite a bit both in terms of additional feature support and detailing in how we handle creating the payjoin transaction proposal

Reply:
This code uses `bpu` and `pj`.